test/com/facebook/buck/go/GoAssumptions.java

```diff
@@ -43,9 +43,9 @@
 
 abstract class GoAssumptions {
   // e.g., "go version go1.9.2 darwin/amd64", "go version go1.9.2 windows/amd64",
-  // "go version go1.2.1 linux/amd64"
+  // "go version go1.8 linux/amd64"
   private static final Pattern VERSION_PATTERN =
-      Pattern.compile("^go version go(\\d+)\\.(\\d+)\\.(\\d+).*$");
```

Comment:
A better fix would be to allow specifying both variants so that you don't need to change this code in future.

Reply:
The new regex allows both variants. The new regex ignores the patch version as there shouldn't be any new feature added as the patch version changes.